### ISSUE TITLE
What is embedding_weight during training?

### ISSUE THREAD

**Issue by YunanWu2168** [LonglongaaaGo/EXE-GAN#8]

**Comment by LonglongaaaGo** [comment]
@YunanWu2168 ,

Thanks for your attention.
The mask_weight is the corresponding confidence weight mask, while the ada_embedding_mask is the corresponding reverse weight mask in our paper.
        # confidence weight mask
        mask_weight = confidence_mask_layer(mask_01)
        # reverse weight mask
        ada_embedding_mask = (1.0- mask_weight)*mask_01
        ada_embedding_mask = (ada_embedding_mask- torch.min(ada_embedding_mask))/(torch.max(ada_embedding_mask) - torch.min(ada_embedding_mask))

**Comment by YunanwuNancy** [comment]
Thank you so much for the reply. If  want to train this model for the size of 512. Do you have any suggestions?

I actually have tried the default hyperparameter values, but the results are not that promising. Thank you again.

**Comment by YunanWu2168** [comment]
Thank you so much for the reply. Actually, If want to train this model for the size of 512. Do you have any suggestions?
I actually have tried the default hyperparameter values, but the results are not that promising. Thank you again.

**Comment by LonglongaaaGo** [comment]
Hi @YunanWu2168 ,
I started to train the 512 model now.
Let you know later for my update.
Thanks!

**Comment by LonglongaaaGo** [comment]
Hi @YunanWu2168 ,
I have updated the new 512x512 pre-trained model, and it now works well. 
The issue appears to have been caused by using masks that were too small for the 512x512 images, resulting in an insufficient masked ratio. 
Since the mask configuration wasn't adjusted for larger images, we have now incorporated the co-mod-gan mask for training 512x512 images.

For further details, please refer to the updated readme file, where you'll find the pre-trained 512x512 model. Additionally, please use the train_largeMask.py file for training 512x512 images.

Thanks.

**Comment by LonglongaaaGo** [comment]
It looks like there are no more questions.
We can re-open it when we are still having some problems.
